package ok.dht.test.slastin.lsm.exception;

public class StorageClosedDaoException extends DaoException {

    public StorageClosedDaoException(Throwable causedBy) {
        super(causedBy);
    }

}
